src/main/java/mate/academy/springboot/swagger/dto/ProductRequestDto.java

```diff
@@ -0,0 +1,10 @@
+package mate.academy.springboot.swagger.dto;
+
+import java.math.BigDecimal;
+import lombok.Data;
+
+@Data

```

Comment:
I think in this case we can use only @Getter (create only getters, and we don't need another)